stdlib/Pkg.version

```diff
@@ -1,2 +1,2 @@
 PKG_BRANCH = master
-PKG_SHA1 = d305e82fd353cb67e8a064800b9972ee1cb7b5e0
+PKG_SHA1 = f9180e48b27a843aeee864db814dce57cb296b9b
```

Comment:
Why is this changing the Pkg sha?

Reply:
It had the wrong SHA1 from what was intended here. This is now the whole commit hash, while previously it only contained the tree hash for this commit. This var is (intentionally) allowed to specify any git object (incl. branch, tag, etc.), but typically it should have the commit SHA, since that's the most general and easiest to work with (e.g. what git usually presents you).

Reply:
But why are the checksums also changed? Shouldn't the commit you set here produce the same tarball?

Reply:
The name here gets inserted by GitHub as part of the tarball file contents (`JuliaLang-Pkg.jl-f9180e4/` vs. `JuliaLang-Pkg.jl-d305e82/`)